### ISSUE TITLE
[KK] Unchecking "Per-Coord" always reverts to Outfit 1's settings, regardless of selected outfit. (v5.2.2)

### ISSUE THREAD

**Issue by Jatts-Art** [ManlyMarco/ABMX#56]
This was noticed in v5.2.2 of ABMX that when you uncheck the "per-coord" option it will reset that bone's settings of all outfits to be based on only the first outfit. According to what was said briefly in the KK server, evidently this is not at all intentional too.

Normally in previous versions, with any outfit selected, what gets apply to all outfits would be the bone settings you are currently looking at and the outfit you have currently selected. So if i were to uncheck the option, then the options I have for the current outfit would then be instantly applied to all other outfits. Copying & pasting bone settings across outfits does not work either, sadly.

I hadn't updated ABMX for a good half year or so, prior to this, and I do not remember which version I had. All I do know is that this bug didn't exist at some point before v5.2.

I've recorded a clip to demonstrate specifically the steps involved for replicating this (converted to gif because video was still too large for github after multiple recording attempts):

<details><summary>Clip (slightly NSFW)</summary>

![ezgif-1-51fd48dfbb](https://github.com/ManlyMarco/ABMX/assets/12752293/643cc4b2-46a0-4cf4-a72a-e8fe45b7f138)

</details>

**Comment by Jatts-Art** [comment]
Okay an update, I went back to test even older versions of ABMX to find out at what point this was working and it turns out............ none of them. Which was obviously not the case for me beforehand, so therefore that means it must be a plugin conflict and zero idea what plugin it could be. I will do a test in-game once again, but this time gonna post an output log afterward.

**Comment by Gylthas** [comment]
This is not the only issue regarding "per-coordinate" reverting settings. Now we have realized that after either toggling "per-coordinate" on/off, reverts to the default state of that bone after changing outfits. If you toggle off a bone that was previously set as "per-coordinate" it toggles back on after changing to a different outfit. And it is also the same the other way around.

This one is not just something annoying. It literally breaks the possibility of making any kind "per-coordinate" edits, unless the bone is already set to that.

**Comment by Jatts-Art** [comment]
Yeah basically this is very screwed up now, one way or another, to the extent that it is definitely not our ends alone. Somewhere along the line, evidently, a combination of updates between ABMX and KK API somehow has presented these issues? Its basically requiring a chain reaction of reverting updates to many things, and we still havent even found the "correct" combination to get it working like it used to. Major bruh moment.
